packages/google_sign_in/google_sign_in_web/example/integration_test/src/jwt_examples.dart

```diff
@@ -38,6 +44,22 @@ const String goodJwtToken =
 const String goodPayload =
     'eyJlbWFpbCI6ImFkdWx0bWFuQGV4YW1wbGUuY29tIiwic3ViIjoiMTIzNDU2IiwibmFtZSI6IlZpbmNlbnQgQWR1bHRtYW4iLCJwaWN0dXJlIjoiaHR0cHM6Ly90aGlzcGVyc29uZG9lc25vdGV4aXN0LmNvbS9pbWFnZT94PS5qcGcifQ';
 
+/// A JWT token with minimal set of predefined values.
+///
+/// 'email': 'adultman@example.com',
+/// 'sub': '123456'
+///
+/// Signed with HS256 and the private key: 'symmetric-encryption-is-weak'
+const String minimalJwtToken =
+    'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.$minimalPayload.UTAe7dpdtFIMwsOqkZkjyjqyHnho5xHCcQylUFmOutM';
```

Comment:
Thanks for adding a test!

I'm glad you managed to create one of these, it's not a trivial process!